jha/src/views/enrolment/SuppBenInfoPage.vue

```diff
@@ -528,7 +528,53 @@ export default {
     return validations;
   },
   methods: {
+    correlateData() {
```

Comment:
Can you rename this method? I think it should be more descriptive. Something like "setEmptyFields" or "setDefaultValues", etc.